src/editor/editor.cpp

```diff
@@ -315,12 +298,18 @@ namespace Editor
             ImGui::Separator();
             ImGui::Text("Tile Set Properties");
 
-
+            ImGui::InputText(" ", image_file_path, IM_ARRAYSIZE(image_file_path), ImGuiInputTextFlags_ReadOnly);
```

Comment:
The idea behind this is that the user could either:
A. enter a filename in the inputText manually, or
B. select the filename with the "Browse" button. 

Then the user can use the "load tileset" button to actually load the image. 

There are two things you can do here:
- Allow user to enter text. It would not be readonly
- Make this just a Text component instead of InputText. Remvoe the "load tileset" button, since it would serve no purpose to have to do two button presses. Upon selection of the image filename, the image would be loaded automatically.